day06/kevin/js/index.js

```diff
@@ -0,0 +1,45 @@
+const endpoint =
+  'https://gist.githubusercontent.com/Miserlou/c5cd8364bf9b2420bb29/raw/2bf258763cdddd704f8ffd3ea9a3e81d25e2c6f6/cities.json';
+
+const select = (identity) => document.querySelector(identity);
+
+const getData = () => fetch(endpoint).then((res) => res.json());
+
+const getFilteredDataByKeyword = (data, keyword = '') => {
+  return !keyword
```

Comment:
try `early return` instead

Reply:
請問 Hello！
這邊用三元運算子寫不算是 early return 嗎？

還是說是因為這樣子寫比較不好讀呢～
感謝哈囉的 review

Reply:
三元不太算 `early reture`且這邊後面做的事情相對較長，建議分開
然後判斷如可樂說的應該為

```suggestion
  if (keyword === '') {
    return []
  }
```

Reply:
OKOK 這樣子我理解了
謝謝哈奇